.github/workflows/publish.yml

```
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  publish-github:
+    name: Publish Package
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')  
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'          
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install
+        run: yarn install --ignore-scripts
+      - name: Publish
+        run: yarn publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

```suggestion
          NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN}}
```
For consistency with other CIs. See for example here: https://github.com/eclipse-cdt-cloud/.eclipsefdn/blob/4d17aa1bc29321c976a319e110d118241af8415e/otterdog/eclipse-cdt-cloud.jsonnet#L137

I believe we need to raise a PR for the above file to add the token. I assume using the same "value" as in above referenced example (the bot thingy).

---

@planger could you take a look?

---

Happy to open a PR. But may need @planger 's approval as project lead anyway.

---

https://github.com/eclipse-cdt-cloud/.eclipsefdn/pull/21